monero-outreach-docs/translations/es/merchants/how_to_accept_monero_es.md

```diff
@@ -0,0 +1,85 @@
+
+En todos estos ejemplos, vamos a decir que estamos vendiendo nuestra limonada, valuada en nuestra moneda local, por $1. Nuestra contabilidad y carga fiscal también estará en la moneda local. Tus condiciones dependerán de donde vivas y cómo está regulado tu tipo de negocio.
+
+### _Puesto de LeMonero_
+
+En nuestro puesto de limonada tenemos una simple caja de cambio y un cuaderno para grabar cada venta, como es común en los mercados de productores, festivales, eventos minoristas emergentes y ventas de artesanías comunitarias donde el equipamiento es limitado. La buena noticia es que aceptar Monero en este tipo de transacciones solo requiere de un móvil.
+
+Para los pagos en Monero, la transacción comienza igual que con el dinero fiat. Preparas la limonada, estableces el total y guardas la transacción en el cuaderno acorde a los requisitos de tu contabilidad.
+
+Un total de $1.00 es convertido a XMR justo antes del pago. La tasa de cambio cambia constantemente. Al momento de la redacción, 1 XMR cuesta $61.40, por lo que una limonada de $1.00 serían 0.0162 XMR.
+
+El cliente necesita tu dirección de Monero en su móvil para pagarte 0.0162 XMR. Copiar y pegar la dirección suele ir bien, pero normalmente se puede escanear con un código QR. El código QR contiene tu dirección de XMR, de forma que la billetera de tu cliente puede escanearla. Tu billetera la genera cuando seleccionas 'recibir', y puedes girar tu móvil hacia tu consumidor para que la escanee. Otra alternativa sería tener tu código QR impreso y así usar únicamente tu billetera para confirmar la transacción.
+
+Tan pronto como el cliente te envíe la transacción, esta es emitida a la red de Monero. Los fondos aparecerán en tu billetera en 4-12 minutos tras la primera confirmación.
+
+Sin embargo, rara vez tenemos que hacer esperar a un cliente, ya que la confianza se puede extender a los clientes de limonada, especialmente a los habituales. Para las escasas personas con grandes pedidos, deberíamos explicarles simplemente que ello tomaría unos pocos minutos.
+
+### _Tienda LeMonero_
+
+Con trabajo duro, nuestro pequeño puesto de limonada se ha convertido en una tienda. Ahora somos un establecimiento fijo. Además de Monero y el efectivo, la tienda acepta tarjetas de crédito y débito y tiene un buen sistema de pagos para grabar las transacciones y administrar el inventario en vez de un cuaderno.
+
+LeMonero tiene una gastos generales elevados, y no podemos permitirnos mantener únicamente pagos en XMR, necesitamos convertir algunos XMR en fiat para pagar las facturas, aunque elegimos liquidarlo manualmente cuando más nos conviene. Todavía necesitamos un procesador de pagos, sin embargo, las comprobaciones de los clientes son fáciles y rápidas.
```

Comment:
I'm not sure if fiat is common known word . I'd change to **dinero tradicional**